2006-06-01      Arnaud Charlet  <charlet@adacore.com>

	* configure: work around libtool bug on ia64 hpux
	See http://bugzilla.gnome.org/show_bug.cgi?id=343485

*** configure.old	Fri May 26 08:34:27 2006
--- configure	Thu Jun  1 05:09:42 2006
***************
*** 11426,11432 ****
  
  	case $host_cpu in
  	hppa*64*|ia64*)
! 	  hardcode_libdir_flag_spec_ld='+b $libdir'
  	  hardcode_direct=no
  	  hardcode_shlibpath_var=no
  	  ;;
--- 11426,11432 ----
  
  	case $host_cpu in
  	hppa*64*|ia64*)
! 	  hardcode_libdir_flag_spec_ld='${wl}+b ${wl}$libdir'
  	  hardcode_direct=no
  	  hardcode_shlibpath_var=no
  	  ;;
